Ms graph user permissions bug fix (#18053)
demisto/content: Packs/MicrosoftGraphUser/Integrations/MicrosoftGraphUser/MicrosoftGraphUser_description.md

> ## Authorize Cortex XSOAR for Azure Active Directory Users

Reviewer: ```suggestion
## Authorize Cortex XSOAR for Azure Active Directory Users (Cortex XSOAR Azure App)
```

> ## Authorize Cortex XSOAR for Azure Active Directory Users (self-deployed configuration)

Reviewer: ```suggestion
## Authorize Cortex XSOAR for Azure Active Directory Users (Self deployed Azure App)
```

There are two different authentication methods for self-deployed configuration: 

> - [client credentials flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow)

Reviewer: ```suggestion
- [Client Credentials flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-client-creds-grant-flow)
```

> - [authorization code flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow).

Reviewer: ```suggestion
- [Authorization Code flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow)
```

> We recommend using the client credentials flow.

Reviewer: ```suggestion
We recommend using the Client Credentials flow.
```

> In order to use the ***msgraph-user-change-password*** command, you must configure with the authorization code flow.

Reviewer: ```suggestion
In order to use the ***msgraph-user-change-password*** command, you must use with the Authorization Code flow.
```

> Note: When using the authorization code flow, make sure the user you authenticate with has the correct roles in Azure AD in order to use the command.

Reviewer: ```suggestion
**Note:** When using the Authorization Code flow, make sure the user you authenticate with has the relevant roles in Azure AD in order to execute the operation.
```

> ### Self deployed configuration with client credentials flow

Reviewer: this and the below section are already documented in our ms auth in xsoar.pan.dev, lets just refer to there
